zstacklib/zstacklib/utils/sanlock.py

```
@@ -1,20 +1,62 @@
+        host_id = lease_struct.host_id
+        r, _, e = repair_delta_lease_if_corrupted(vg_name, host_id)
+    else:
+        resource_name = lease_struct.resource_name
+        r, _, e = repair_paxos_lease(vg_name, resource_name, offset)
+
+    if r != 0:
+        return "failed to repair sanlock lease, err %s" % e
+    return None
+
+
+def request_mn_repair_metadata(vg_name, specify_host_by_id=False, **lease_struct):
```

Pylint (3.3.7)

[refactor] Either all return statements in a function should return an expression, or none of them should.

(R1710)